src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
+const tree = document.querySelector('.tree');
+// const li = document.querySelector('li');
+
+const wrapSpan = (node) => {
+  for (const li of node.querySelectorAll('li')) {
+    const text = li.firstChild;
+
+    const span = document.createElement('span');
+
+    span.textContent = text.textContent;
```

Comment:
Instead of `text.textContent`, you should use `text.nodeValue` to get the text content of a text node. This ensures that you are correctly accessing the text of the first child node, which is expected to be a text node.